service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/NotificationHandlerTest.java

```diff
@@ -536,6 +542,34 @@ void shouldNotifyLocalAuthorityOfIssuedStandardDirectionsOrder() throws IOExcept
             eq("12345"));
     }
 
+    @Test
+    void shouldNotifyAdminOfPlacementApplicationUpload() throws Exception {
+        CallbackRequest callbackRequest = callbackRequest();
+        CaseDetails caseDetails = callbackRequest().getCaseDetails();
+        final Map<String, Object> expectedParameters = getExpectedPlacementNotificationParameters();
+
```

Comment:
can remove one of these whitespace lines

Reply:
whoops, good spot.